cdap-data-fabric/src/test/java/co/cask/cdap/data2/metadata/writer/BasicLineageWriterTest.java

```diff
@@ -58,15 +62,16 @@ public void testWrites() throws Exception {
     LineageWriter lineageWriter = new BasicLineageWriter(lineageStore);
 
     // Define entities
-    Id.Program program = Id.Program.from("default", "app", ProgramType.FLOW, "flow");
-    Id.Stream stream = Id.Stream.from("default", "stream");
-    Id.Run run1 = new Id.Run(program, RunIds.generate(10000).getId());
-    Id.Run run2 = new Id.Run(program, RunIds.generate(20000).getId());
+    ProgramId program = ProgramId.fromIdParts(Arrays.asList(NamespaceId.DEFAULT.getNamespace(), "app",
+                                              ProgramType.FLOW.getPrettyName(), "flow"));
+    StreamId stream = StreamId.fromIdParts(Arrays.asList(NamespaceId.DEFAULT.getNamespace(), "stream"));
+    Id.Run run1 = new Id.Run(program.toId(), RunIds.generate(10000).getId());
```

Comment:
it may be better to use ``ProgramRunId`` here, and use ``toId`` in its usages. This could become more compact then - ``program.run(RunIds.generate(10000).getId())``